sources/packages/forms/src/form-definitions/studentdependantsappealparttime.json

```diff
@@ -0,0 +1,933 @@
+                                            "defaultValue": "",
+                                            "protected": false,
+                                            "fieldSet": false,
+                                            "persistent": true,
+                                            "hidden": false,
+                                            "clearOnHide": true,
+                                            "validate": {
+                                                "required": true,
+                                                "custom": "",
+                                                "customPrivate": false
+                                            },
+                                            "type": "radio",
+                                            "labelPosition": "top",
+                                            "optionsLabelPosition": "right",
+                                            "tags": [],
+                                            "conditional": {
+                                                "show": "",
+                                                "when": null,
+                                                "eq": ""
+                                            },
+                                            "properties": {},
+                                            "lockKey": true,
+                                            "customClass": "\"customClass\": \"label-bold\""
+                                        },
+                                        {
+                                            "clearOnHide": false,
+                                            "key": "addDependantsUploadPanel",
+                                            "input": false,
+                                            "title": "Upload panel",
+                                            "theme": "default",
+                                            "tableView": false,
+                                            "components": [
+                                                {
+                                                    "key": "addDependantsUploadHtml",
+                                                    "label": "HTML",
+                                                    "input": false,
+                                                    "tag": "h3",
+                                                    "attrs": [
+                                                        {
+                                                            "value": "",
+                                                            "attr": ""
+                                                        }
+                                                    ],
+                                                    "className": "category-header-medium primary-color",
+                                                    "content": "Upload proof of your dependants income tax return for 20[XX]",
```

Comment:
I consider this to be NOT part of the ticket. But can ask business
Does the value of the tax year needs to be precise here and student application replacing XX??

![image](https://github.com/bcgov/SIMS/assets/54600590/b71186da-11cf-4a48-b0e6-4dd562cd69c2)

Reply:
@dheepak-aot Not technically part of this ticket. Will need to be specific based on the program year of the application the change request is for.